pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -311,6 +311,12 @@ func (r *ReconcileOperConfig) Reconcile(ctx context.Context, request reconcile.R
 	// Fill all defaults explicitly
 	network.FillDefaults(&newOperConfig.Spec, prev, mtu)
 
+	klog.Infof("IPsec: bootstrap: ipsecConfig: %v", newOperConfig.Spec.DefaultNetwork.OVNKubernetesConfig.IPsecConfig)
+	klog.Infof("IPsec: bootstrap: prev: %v", prev.DefaultNetwork.OVNKubernetesConfig.IPsecConfig)
```

Comment:
Can you explain the purpose of the bootstrap logic to me? I'm trying to understand what it's trying to achieve/what problem it solves

So far I've got to:
* Every time the operator starts, it creates a default `Network` object with all of the defaults populated
* It then applies this (using server side apply?) to the cluster?

How does this work for values that already exist? The `FillDefaults` doesn't modify anything that doesn't already exist?

Why do we need this up conversion? The API server should fill out the defaults for you if any have changed